README.md

```diff
@@ -72,6 +72,19 @@ intended for itself.
 
 The node also still allows access to the [message subsystem](#message-system).
 
+## Configuration
```

Comment:
Lets make it clear here that the config file is optional, and that not providing one and not having one in the config directory is fine.